src/main/java/cn/nukkit/entity/EntityHuman.java

```diff
@@ -144,8 +142,8 @@ protected void initEntity() {
                 if (skinTag.contains("GeometryData")) {
                     newSkin.setGeometryData(new String(skinTag.getByteArray("GeometryData"), StandardCharsets.UTF_8));
                 }
-                if (skinTag.contains("AnimationData")) {
-                    newSkin.setAnimationData(new String(skinTag.getByteArray("AnimationData"), StandardCharsets.UTF_8));
+                if (skinTag.contains("SkinAnimationData")) {
```

Comment:
Any reason why these NBT tags have been renamed? Anyone who depends on loading the skins of players within a plugin will lose the data because you've changed the names without any kind of backward compatibility. These are also internal Nukkit names so they don't have to be exactly the same as vanilla.

Reply:
Didn't think about plugins. Otherwise the data wouldn't have existed because of incorrect loading on player login.